### ISSUE TITLE
Display codewars prereq info: Kyu level

### ISSUE THREAD

**Issue by BartBucknill** [ameliejyc/prereq-check#2]
As a selection panel member I can see whether the applicant has attained 5 Kyu.
As an applicant I can see whether I have attained 5 Kyu.

Acceptance criteria:

- [x] Display whether applicant has attained 5 Kyu
- [x] Display page should look like designed summary page (image to follow)

![summary page](https://user-images.githubusercontent.com/14337958/30059069-7f000886-9235-11e7-8b4f-75b53c1a1407.png)

**Comment by dangerdak** [comment]
blocked by #16

**Comment by dangerdak** [comment]
@astroash relating to cea3e68.
 I think we were handling errors incorrectly: https://github.com/request/request-promise#get-a-rejection-only-if-the-request-failed-for-technical-reasons (this suggests that the default behaviour is for the catch block to be called if the response status code is 404, like it is when we provide an incorrect username)

So I've moved the logic to deal with the case of an invalid user outside of the getKyu function, and it's checked in the router instead. 

Happy to roll back the commit if you feel like I've made stuff worse/more confusing.
